controllers/dspipeline_controller.go

```diff
@@ -573,6 +574,9 @@ func (r *DSPAReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				return reconcileRequests
 			})).
 		// TODO: Add watcher for ui cluster rbac since it has no owner
+		WithOptions(controller.Options{
+			MaxConcurrentReconciles: 10,
```

Comment:
should this also be a variable? (no magic numbers!)

Reply:
fixed